src/operation/get_more/mod.rs

```diff
@@ -15,7 +15,7 @@ use crate::{
     Namespace,
 };
 
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
This is needed so that we can send owned GetMore operations to `execute_operation` as described above.